WordPress/gutenberg
# Global styles revisions: refactor sticky pagination footer to use CSS

## What?
A follow up to:

- https://github.com/WordPress/gutenberg/pull/56799

Let's fix the pagination to the bottom of the sidebar panel using CSS.

We can now delete the Slot Container, which is not used elsewhere.

> [!WARNING]
> Because the footer now sits within a scrollable container, the width of the pagination will be affected by the browser's scrollbar. See the screencasts below. So, one option is to forget about this PR.

## Why?
CSS.

## How?
Making the footer sticky. 

And relying on the convenient fact that `.edit-site-global-styles-sidebar__navigator-screen` has a `display: flex` rule.

## Testing Instructions
Open up the site editor, ensuring you have plenty of Global styles revisions (more than 10 to enable pagination).

Check that the pagination:

1. Works
2. Sticks to the bottom.

## Screenshots or screencast 

### Trunk
https://github.com/WordPress/gutenberg/assets/6458278/8be2af80-f604-4952-86df-96f74fb90a5e


### This PR

https://github.com/WordPress/gutenberg/assets/6458278/b69dcb0d-2443-4d24-bbaa-32c27448a106
